tests/test_moodle.py

```diff
@@ -382,3 +382,20 @@ def test_get_users_by_course(mocker):
     )
 
     assert users_json == [{}]
+
+
+def test_get_user_uuids(mocker):
+    session_mock = mocker.Mock()
+    session_mock.get.return_value.json.return_value = [{}]
+    client = moodle.MoodleClient(
+        session_mock, TEST_MOODLE_URL, TEST_MOODLE_TOKEN
+    )
+    client.get_user_uuids([])
```

Comment:
We may want to also add a test case where we pass a specific ID. I think the current implementation may actually have a bug per some of my other comments :man_shrugging: .